### DISCUSSION TITLE
Can the new version storybook support react 16?

### DISCUSSION THREAD

**Discussion by dengxiao0509** [storybookjs/storybook#25637] [Help] [answered]
### Summary

hi,can the new version storybook support react 16?
my project and components use react 16, but it seems that storybook uses react 18 to render the UI.

**Comment by valentinpalkovic** [comment, marked as answer]
Hi @dengxiao0509,

We are still supporting React 16. Only Storybook's Manager UI uses React 18. Your components are treated encapsulated, though :)
